.github/workflows/on-pullrequest.yml

```diff
@@ -73,10 +73,10 @@ jobs:
         if: ${{ steps.modules-cache.outputs.cache-hit != 'true' }}
         run: npm install
 
-      - name: Dependency Review ("Dependabot on PR")
-        # This action can only run in pull requests
-        if: ${{ github.event.pull_request }}
-        uses: actions/dependency-review-action@a9c83d3af6b9031e20feba03b904645bb23d1dab # v1.0.2
+      # - name: Dependency Review ("Dependabot on PR")
```

Comment:
I had to comment out dependabot checks temporarily here. It's dying on the xml2js issue in MetaEd, which has already been flagged by us as ok to live with. For some reason this action still flags it. It would be very risky try to upgrade in MetaEd, given we'd have to revert MetaEd to MyGet and bump/republish there.